2025-02-25 17:28:56,621 - INFO - Starting Application...
2025-02-25 17:28:56,626 - INFO - Validating dataset_one with Pydantic.
2025-02-25 17:29:00,196 - ERROR - 1 validation error for EmployeeExpertiseCalls
calls_made
  Input should be greater than or equal to 0 [type=greater_than_equal, input_value=-54, input_type=int]
    For further information visit https://errors.pydantic.dev/2.10/v/greater_than_equal | Row: {'id': 1, 'area': 'IT', 'calls_made': -54, 'calls_successful': 41} 
2025-02-25 17:29:00,196 - ERROR - 'NoneType' object has no attribute 'lower' | Row: {'id': 6, 'area': None, 'calls_made': 29, 'calls_successful': 24} 
2025-02-25 17:29:00,204 - WARNING -  A total of 2 invalid records excluded from dataset dataset_one
2025-02-25 17:29:00,304 - INFO - Validating dataset_two with Pydantic.
2025-02-25 17:29:00,523 - WARNING -  A total of 0 invalid records excluded from dataset dataset_two
2025-02-25 17:29:00,626 - INFO - Processing outputs.
2025-02-25 17:29:00,853 - INFO - Output it_data is being written to C:\Users\be_al\Github\abn_amro\src\sales_data\..\..\data\output/it_data
2025-02-25 17:29:14,686 - INFO - Output marketing_address_info is being written to C:\Users\be_al\Github\abn_amro\src\sales_data\..\..\data\output/marketing_address_info
2025-02-25 17:29:25,928 - INFO - Closing Application...
2025-02-25 17:29:25,939 - INFO - Closing down clientserver connection
2025-02-25 18:36:39,875 - INFO - Starting Application...
2025-02-25 18:36:39,877 - INFO - Validating dataset_one with Pydantic.
2025-02-25 18:36:47,475 - ERROR - 1 validation error for EmployeeExpertiseCalls
calls_made
  Input should be greater than or equal to 0 [type=greater_than_equal, input_value=-54, input_type=int]
    For further information visit https://errors.pydantic.dev/2.10/v/greater_than_equal | Row: {'id': 1, 'area': 'IT', 'calls_made': -54, 'calls_successful': 41} 
2025-02-25 18:36:47,477 - ERROR - 'NoneType' object has no attribute 'lower' | Row: {'id': 6, 'area': None, 'calls_made': 29, 'calls_successful': 24} 
2025-02-25 18:36:47,479 - WARNING - A total of 2 invalid records excluded from dataset dataset_one
2025-02-25 18:36:47,614 - INFO - Validating dataset_two with Pydantic.
2025-02-25 18:36:47,938 - INFO - There were no records excluded from dataset dataset_two
2025-02-25 18:36:48,025 - INFO - Processing outputs.
2025-02-25 18:36:48,187 - INFO - Processing agents selling IT products. Total rows:DataFrame[area: string, calls_made: bigint, calls_successful: bigint, id: bigint, address: string, emp_id: bigint, name: string, sales_amount: double]
2025-02-25 18:37:34,119 - INFO - Successfully processed top 100 agents selling IT products: 100 records
2025-02-25 18:37:34,119 - INFO - Output it_data is being written to data/output/it_data
2025-02-25 18:37:52,800 - INFO - Processing marketing_adress_info output.
2025-02-25 18:37:52,867 - INFO - Output marketing_address_info is being written to data/output/marketing_address_info
2025-02-25 18:38:10,946 - INFO - Closing Application...
2025-02-25 18:38:10,964 - INFO - Closing down clientserver connection
2025-02-25 19:02:18,296 - INFO - Starting Application...
2025-02-25 19:02:18,296 - INFO - Validating dataset_one with Pydantic.
2025-02-25 19:02:25,655 - ERROR - 1 validation error for EmployeeExpertiseCalls
calls_made
  Input should be greater than or equal to 0 [type=greater_than_equal, input_value=-54, input_type=int]
    For further information visit https://errors.pydantic.dev/2.10/v/greater_than_equal | Row: {'id': 1, 'area': 'IT', 'calls_made': -54, 'calls_successful': 41} 
2025-02-25 19:02:25,655 - ERROR - 'NoneType' object has no attribute 'lower' | Row: {'id': 6, 'area': None, 'calls_made': 29, 'calls_successful': 24} 
2025-02-25 19:02:25,663 - WARNING - A total of 2 invalid records excluded from dataset dataset_one
2025-02-25 19:02:25,868 - INFO - Validating dataset_two with Pydantic.
2025-02-25 19:02:26,368 - INFO - There were no invalid records found in dataset_two
2025-02-25 19:02:26,481 - INFO - Processing outputs sequentially...
2025-02-25 19:02:26,622 - INFO - Processing it_data output...
2025-02-25 19:02:47,716 - INFO - Successfully processed top 100 agents selling IT products.
2025-02-25 19:02:47,716 - INFO - Output it_data is being written to data/output/it_data
2025-02-25 19:03:04,950 - INFO - Processing marketing_adress_info output...
2025-02-25 19:03:05,002 - INFO - Output marketing_address_info is being written to data/output/marketing_address_info
2025-02-25 19:03:21,287 - INFO - Closing Application...
2025-02-25 19:03:21,301 - INFO - Closing down clientserver connection
2025-02-25 19:09:04,756 - INFO - Starting Application...
2025-02-25 19:09:04,757 - INFO - Validating dataset_one with Pydantic.
2025-02-25 19:09:10,874 - ERROR - 1 validation error for EmployeeExpertiseCalls
calls_made
  Input should be greater than or equal to 0 [type=greater_than_equal, input_value=-54, input_type=int]
    For further information visit https://errors.pydantic.dev/2.10/v/greater_than_equal | Row: {'id': 1, 'area': 'IT', 'calls_made': -54, 'calls_successful': 41} 
2025-02-25 19:09:10,874 - ERROR - 'NoneType' object has no attribute 'lower' | Row: {'id': 6, 'area': None, 'calls_made': 29, 'calls_successful': 24} 
2025-02-25 19:09:10,884 - WARNING - A total of 2 invalid records excluded from dataset dataset_one
2025-02-25 19:09:11,067 - INFO - Validating dataset_two with Pydantic.
2025-02-25 19:09:11,511 - INFO - There were no invalid records found in dataset_two
2025-02-25 19:09:11,609 - INFO - Processing outputs sequentially...
2025-02-25 19:09:11,734 - INFO - Processing output it data...
2025-02-25 19:09:33,443 - INFO - Successfully processed top 100 agents selling IT products.
2025-02-25 19:09:33,443 - INFO - Writing Output it_data to location: data/output/it_data
2025-02-25 19:09:50,420 - INFO - Processing output marketing address info...
2025-02-25 19:09:50,463 - INFO - Writing Output marketing_address_info to location: data/output/marketing_address_info
2025-02-25 19:10:07,098 - INFO - Started processing department breakdown...
2025-02-25 19:10:26,609 - INFO - Writing Output department_breakdown to location: data/output/department_breakdown
2025-02-25 19:10:43,569 - INFO - Closing Application...
2025-02-25 19:10:43,589 - INFO - Closing down clientserver connection
2025-02-25 19:17:50,593 - INFO - Starting Application...
2025-02-25 19:17:50,602 - INFO - Validating dataset_one with Pydantic.
2025-02-25 19:17:56,935 - INFO - There were no invalid records found in dataset_one
2025-02-25 19:17:57,066 - INFO - Validating dataset_two with Pydantic.
2025-02-25 19:17:57,435 - INFO - There were no invalid records found in dataset_two
2025-02-25 19:17:57,543 - INFO - Processing outputs sequentially...
2025-02-25 19:17:57,701 - INFO - Processing output it data...
2025-02-25 19:18:19,698 - INFO - Successfully processed top 100 agents selling IT products.
2025-02-25 19:18:19,698 - INFO - Writing Output it_data to location: data/output/it_data
2025-02-25 19:18:36,669 - INFO - Processing output marketing address info...
2025-02-25 19:18:36,730 - INFO - Writing Output marketing_address_info to location: data/output/marketing_address_info
2025-02-25 19:18:52,485 - INFO - Started processing department breakdown...
2025-02-25 19:19:11,695 - INFO - Writing Output department_breakdown to location: data/output/department_breakdown
2025-02-25 19:19:33,633 - INFO - Closing Application...
2025-02-25 19:19:33,647 - INFO - Closing down clientserver connection
